Tests/SwiftDocCTests/Semantics/SnippetTests.swift

```diff
@@ -87,4 +87,18 @@ class SnippetTests: XCTestCase {
         XCTAssertNotNil(snippet)
         XCTAssertTrue(problems.isEmpty)
     }
+
+    func testCommentResolves() throws {
```

Comment:
(Sorry maybe this is still a WIP – but it looks like it's a copy/paste of the existing `testLinkResolves` test.)

Reply:
oops this was old... thanks for catching this!